utility/WiFiStream.h

```diff
@@ -124,6 +149,7 @@ class WiFiStream : public Stream
     return result;
   }
 
+#ifndef ESP8266
```

Comment:
I could not find matching methods for WEP security in the ESP8266/arduino project. As far a I know the ESP8266 supports WEP. I will have another look into this. 

The other begin() functions also use the local IP only definition without netmask and gateway that is not supported by the ESP8266. All these functions are not used by the example.